modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -162,6 +170,20 @@ def find_bgs_claim!(claim_id:)
           raise
         end
 
+        def find_tracked_items!(claim_id)
+          return if claim_id.blank?
+
+          local_bgs_service.find_tracked_items(claim_id)[:dvlpmt_items] || []
+        rescue Savon::SOAPFault => e
+          # the ebenefits service raises an exception if a claim tracked items are not found,
+          # so catch the exception here and return a 404 instead
+          if e.message.include?("No tracked items found for #{claim_id}")
+            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim tracked items not found')
+          end
+
+          raise
```

Comment:
Probably fine for now, but it'd be nice to have more error handling down the road as I'm sure we'll find more errors (I.E. connection timeout or a non-200 response).